apps/spark/src/test/java/com/linkedin/openhouse/catalog/e2e/SparkSchemaEvolutionTest.java

```diff
@@ -0,0 +1,86 @@
+package com.linkedin.openhouse.catalog.e2e;
+
+import com.linkedin.openhouse.jobs.spark.Operations;
+import com.linkedin.openhouse.tablestest.OpenHouseSparkITest;
+import java.util.Arrays;
+import java.util.List;
+import org.apache.iceberg.Schema;
+import org.apache.iceberg.Table;
+import org.apache.iceberg.spark.SparkSchemaUtil;
+import org.apache.iceberg.types.Types;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.RowFactory;
+import org.apache.spark.sql.SparkSession;
+import org.apache.spark.sql.types.DataTypes;
+import org.apache.spark.sql.types.Metadata;
+import org.apache.spark.sql.types.StructField;
+import org.apache.spark.sql.types.StructType;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class SparkSchemaEvolutionTest extends OpenHouseSparkITest {
+
+  @Test
+  void testEndUserSchemaEvolution() throws Exception {
```

Comment:
can you also add another test which has 2 subsequent evolutions. That will help validate that the evolved schema from 1st evolution is not corrupted.

Reply:
I added a subsequent step to do one more evolution, although I am not sure if that's very useful, since the purpose for the change is to let OH server accept schema "as-it-is" instead of honoring some of the contracts, which I believe is the one you wanted to validate with by "That will help validate that the evolved schema from 1st evolution is not corrupted."